PD-PiotrKurzydym/style.css

```diff
@@ -0,0 +1,190 @@
+body {
+  background-image: url("https://upload.wikimedia.org/wikipedia/commons/thumb/c/ce/Star_wars2.svg/2560px-Star_wars2.svg.png");
+}
+
+.active,
+.btn:hover {
+  background-color: #666;
+  color: white;
+}
+#category {
+  display: flex;
+  text-align: center;
+  justify-content: center;
+  color: #24292e;
+  font-family: -apple-system, system-ui, "Segoe UI", Helvetica, Arial,
+    sans-serif, "Apple Color Emoji", "Segoe UI Emoji";
+  font-size: 16px;
+  font-weight: 500;
+}
+#search {
+  position: absolute;
+  right: 5%;
+}
+.header,
+#tablediv {
+  display: inline-block;
+  text-align: center;
+  position: relative;
+  left: 50%;
+  transform: translateX(-50%);
+  margin-top: 10px;
+  /* appearance: none; */
+  background-color: #fafbfc;
+  border: 1px solid rgba(27, 31, 35, 0.15);
+  box-shadow: rgba(27, 31, 35, 0.04) 0 1px 0,
+    rgba(255, 255, 255, 0.25) 0 1px 0 inset;
+  box-sizing: border-box;
+  color: #24292e;
+  font-family: -apple-system, system-ui, "Segoe UI", Helvetica, Arial,
```

Comment:
Możesz zadeklarować font-family dla całego dokumentu. Jest to właściwość która się dziedziczy.